data/remnant missions.txt

```diff
@@ -1252,3 +1252,236 @@ mission "Remnant: Expanded Horizons Quarg 3"
 			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he says while tapping the databanks, "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
 			`	Upon arriving at <planet> they quickly thank you, hand you <payment>, and run back to their offices to finish prepping their material to publish and distribute.`
 
+
+
+mission "Remnant: Broken Jump Drive 1"
+	name "Broken Jump Drive Delivery"
+	description "You have been asked to deliver a broken jump drive to the researchers on <planet>."
+	minor
+	landing
+	to offer
+		has "remnant blood test pure"
+	source
+		government "Remnant"
+		attributes outfitter
+	destination
+		government "Remnant"
+		attributes outfitter
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`As your ship the <ship> settles onto its designated landing pad, you see someone emerge from the nearby outfitter and quickly make their way to you. He waits a safe distance back, then approach once you cycle the hatch and disembark. "Greetings, Captain <last>," trills the Remnant. "We saw the news that the newcomer among us was tracking down an erratic Korath ship. When you came in to land, we saw on the scans that you salvaged its malfunctioning jump drive. Could we take it, to study?"`
```

Comment:
> As your ship the <ship> settles onto its designated landing pad,

As your ship, the <ship>, settles onto its designated landing pad,

Reply:
> He waits a safe distance back, then approach once you cycle the hatch and disembark

The man waits at a safe distance before approaching once you've cycle the hatch and disembark

Reply:
> "Greetings, Captain <last>," trills the Remnant. "We saw the news that the

IMO, would rather have it as "Greetings, Captain <last>," trills the Remnant, "we saw the news that the

Reply:
>Could we take it, to study?"`

Could we take it for study?"`

Reply:
Done. Thanks!